UI/window-basic-main.cpp

```diff
@@ -249,6 +249,7 @@ OBSBasic::OBSBasic(QWidget *parent)
 #endif
 
 	setAttribute(Qt::WA_NativeWindow);
+	setStyle(new OBSIgnoreWheelProxyStyle());
```

Comment:
Out of curiosity, why is it set here instead of in OBSApp?

Reply:
No real reason, mostly because the stylesheet is also set there.
Clarified that a bit by moving it directly above the stylesheet call.

Reply:
Maybe a better idea would to have it in the InitTheme() function in obs-app.cpp, so theming stuff can be in one place. I don't think we even use the setStyleSheet() in window-basic-main.cpp, as it seems to be old code before custom stylesheets were introduced.

Reply:
moved it to OBSApp::InitTheme